galasa-parent/galasa-boot/src/main/java/dev/galasa/boot/Launcher.java

```diff
@@ -561,4 +565,21 @@ private void commandLineError(String message) {
                         + "Example Resource Management arguments: --obr infra.obr --obr test.obr --resourcemanagement");
         System.exit(-1);
     }
+
+    public boolean validateJavaLevel(){
+        String version = System.getProperty("java.version");
```

Comment:
If you passed SYstem.getProperty(...) in as a parameter, you could unit test this method in isolation.

Reply:
Yes - the work to get the Environment class is in a prior PR so once that has merged I can add in here